crates/ide-assists/src/handlers/inline_type_alias.rs

```diff
@@ -62,16 +66,12 @@ pub(crate) fn inline_type_alias_uses(acc: &mut Assists, ctx: &AssistContext<'_>)
             let mut inline_refs_for_file = |file_id, refs: Vec<FileReference>| {
                 builder.edit_file(file_id);
 
-                let path_types: Vec<ast::PathType> = refs
-                    .into_iter()
-                    .filter_map(|file_ref| match file_ref.name {
-                        ast::NameLike::NameRef(path_type) => {
-                            path_type.syntax().ancestors().nth(3).and_then(ast::PathType::cast)
-                        }
-                        _ => None,
-                    })
-                    .collect();
+                let (path_types, path_type_uses) =
+                    split_refs_and_uses(builder, refs, |path_type| {
+                        path_type.syntax().ancestors().nth(3).and_then(ast::PathType::cast)
+                    });
 
+                path_type_uses.iter().for_each(remove_path_if_in_use_stmt);
```

Comment:
This fails, we are looking explicilty for `ast::PathType` usages, but in imports there are no `ast::PathType`s, only `ast::Path`s

Reply:
![image](https://user-images.githubusercontent.com/16398479/187290787-99729438-69ca-4450-b2fb-5cf6474b2ce7.png)

I think the types are correct ;-) did the variable name confuse you?

Reply:
Ah apoligies, I misunderstood how `split_refs_and_uses` worked, yes this is correct then